dashboard/src/components/EventHeader.vue

```diff
@@ -100,4 +95,37 @@ const chapter = createDocumentResource({
   fields: ['*'],
   auto: true,
 })
+
+const isEvent = computed(() => props.form.data?.doctype === 'Event')
+
+function getBadgeTheme() {
+  if (isEvent.value) {
+    switch (props.event.status) {
+      case 'Approved':
+        return 'blue'
```

Comment:
We are removing `Approved` right? 

Status would be the following:
- Draft
- Live
- Concluded
- Cancelled

Reply:
Wasnt sure when we are removing approved. So kept it in, will change